modules/h2/src/main/java/org/h2/command/dml/SelectGroups.java

```diff
@@ -135,6 +143,8 @@ public void remove() {
             cursor.remove();
             currentGroupByExprData = null;
             currentGroupRowId--;
+
+            onUpdate(curEntry.getKey(), curEntry.getValue(), null);
```

Comment:
is it possible NPE here?

Reply:
NPE is impossible here as next() method is always called before this.
Moreover, Cursor.remove will fails with IllegalStateException before NPE happens.